keps/sig-cluster-lifecycle/kubeadm/20190424-kubeadm-for-windows.md

```diff
@@ -93,19 +97,15 @@ The motivation for this KEP is to provide a tool to allow users to take a Window
 
 ### Goals
 
-* Create and maintain a Powershell script to install and run Kubernetes prerequisites on Windows
-* Support kubeadm join for Windows
-* Support kubeadm reset for Windows
-* Create and maintain a Powershell script to deploy Windows CNIs
+* Create and maintain a Powershell script to install kubelet, kubeadm and [wins](https://github.com/rancher/wins/)
+* Support kubeadm join, reset and upgrade
+* Provide DaemonSets to run kube-proxy and Flannel
 
 ### Non-Goals
 
 * Installing the Container Runtime (e.g. Docker or containerd)
-* Implement kubeadm init for Windows (at this time)
-* Implement kubeadm join --control-plane for Windows (at this time)
-* Supporting upgrades using kubeadm upgrade for Windows (to be revisited for Beta)
-* Running kube-proxy as a DaemonSet on Windows (to be revisited for Beta)
-* Running Flannel as a DaemonSet on Windows (to be revisited for Beta)
+* Implement kubeadm init
+* Implement kubeadm join --control-plane
```

Comment:
SGTM, for the time being.

a side question here:
with `wins` can we run the control-plane components in static Pods - are there remaining blockers for that?

Reply:
I've never actually tried, I've always heard there were problems running the control plane on Windows but never investigated what they were

Reply:
i know that e.g. the controller-manager and apiserver need the `ca-certificates` Linux distro package, but other than that i'm wondering whether there is something critical / blocking that cannot be solved with mounts and path re-config.

Reply:
i might try this at some point.